### DISCUSSION TITLE
Delayed calibrations after locking screen

### DISCUSSION THREAD

**Discussion by WojSkwie** [Navid200/xDrip#144] [Testing]
Per Navid request I am creating a separate discussion for this problem of delayed calibrations.

The most important information that I unfortunately omitted in previous post is that it started to happen when I bought a new phone:
Samsung S7 -> Samsung A52s 5G
Android 8.0 -> Android 11

Copied all the setting via QR code.

Reproduction steps:
1. Enter a blood glucose value for automatic calibrations.
2. Lock the screen on the phone.
3. Wait more that the amount of minutes necessary for calibration to happen (e.g. 30 minutes).
4. Unlock the phone.

Calibrations uses the value entered before but applies it on the moment the screen is unlocked.

Screenshots:
![image](https://user-images.githubusercontent.com/34103085/144765872-4a3d4867-d352-4116-a1f1-b44c9256c139.png)
![image](https://user-images.githubusercontent.com/34103085/144765879-55707c4c-5cd3-4544-b187-d481e2dbc4c6.png)
![image](https://user-images.githubusercontent.com/34103085/144765890-2a47fe29-f0b6-4c2a-a8a7-73783b7155f7.png)
![image](https://user-images.githubusercontent.com/34103085/144765897-505bc709-bfdc-468b-b81d-988dfb361f5c.png)
![image](https://user-images.githubusercontent.com/34103085/144765904-7e17c47f-1b5b-4923-a5ef-ba6d76ac2e99.png)
![image](https://user-images.githubusercontent.com/34103085/144765910-846c8450-042d-4860-92b6-4346887ec6ed.png)
![image](https://user-images.githubusercontent.com/34103085/144765926-517195ac-23c0-42e7-bad1-7e12a36040cd.png)

(Btw. Deleting all info in the old thread about this issue)

**Comment by Navid200** [comment]
@WojSkwie 
You can open an issue: https://github.com/NightscoutFoundation/xDrip/issues
Please include a link to this thread.
I would include automatic calibration in the title like Delayed automatic calibration after locking screen.
